avocado/plugins/replay.py

```diff
@@ -97,10 +97,18 @@ def run(self, args):
 
         view = output.View()
 
-        if args.remote_hostname is not None:
-            msg = "Currently we don't replay jobs in remote hosts."
-            view.notify(event='error', msg=(msg))
-            sys.exit(exit_codes.AVOCADO_JOB_FAIL)
+        err = None
+        if args.replay_teststatus and args.multiplex_files:
+            err = "Option --replay-test-status is incompatible with "\
+                  "--multiplex-files."
+        elif args.replay_teststatus and args.url:
+            err = "Option --replay-test-status is incompatible with "\
+                  "informing new tests in command line."
+        elif args.remote_hostname:
+            err = "Currently we don't replay jobs in remote hosts."
+        if err:
```

Comment:
This is **really, really minor**, but the idiom `if err is not None` is preferred.